src/containers/navbar/NavBar.js

```diff
@@ -1,58 +1,33 @@
+import { useState } from 'react'
 import { Link } from 'react-router-dom'
 import { routes } from '~/constants/routes'
 import { useTranslation } from 'react-i18next'
 import { Typography, Box, Button, IconButton, List, ListItem  } from '@mui/material'
 
 import Logo from '~/containers/logo/Logo'
+import Sidebar from '~/containers/sidebar/Sidebar'
 import LanguageIcon from '@mui/icons-material/Language'
 import MenuIcon from '@mui/icons-material/Menu'
-
+import { style } from '~/containers/navbar/navbar.style'
 import PropTypes from 'prop-types'
 
-const style = {
-  header: {
-    display: 'flex',
-    justifyContent: 'space-between',
-    padding: '0px',
-    margin: { xs: '0', xl: 'auto' }, 
-    maxWidth: 'xl',
-    width: { xl: '100%' },
-  },
-  navList: {
-    display: { xs: 'none', md: 'flex' },
-    alignItems: 'center',
-  },
-  navItem: {
-    paddingLeft: '0',
-    width: 'auto',
-    '&::after': {
-      content: '"/"',
-      padding: '0 0 3px 20px',
-    },
-    '&:last-child::after': {
-      content: '""'
-    },
-  },
-  iconBox: {
-    display: 'flex',
-    alignItems: 'center',
-  },
-  menuIcon: {
-    display: { md: 'none' },
-    marginRight: '11px',
-  }
-}
-
 const Navbar = ({ navigationItems, children }) => {
 
   const { t } = useTranslation()
-  
+  const [isOpen, setIsOpen] = useState(false)
+
+  const openSidebar = () => setIsOpen(true)

```

Comment:
I suggest creating one method and passing neither you want to set it to true or false, or rename state ti isSideBarActive, setIsSidebarActive, and use directly setter without methods.